Sources/MagicSDK/Modules/User/UserModule.swift

```diff
@@ -120,4 +120,17 @@ public class UserModule: BaseModule {
             showSettings(response: promiseResolver(resolver))
         }
     }
+    /**
+            recoverAccount
+     */
+    public func recoverAccount(response: @escaping Web3ResponseCompletion<String>) {
+        let request = BasicRPCRequest(method: UserMethod.magic_auth_recover_account.rawValue, params: [])
+        self.provider.send(request: request, response: response)
+    }
+    
+    public func recoverAccount() -> Promise<String> {
```

Comment:
RecoverAccount will take in an object `{ email }` and return with true if recovery is successful. Otherwise, an error will be thrown.